### ISSUE TITLE
Better lighting?

### ISSUE THREAD

**Issue by Peque** [Peque/sphinxcadquery#7]
Have a look at [three.js examples](https://threejs.org/examples/).

Can we improve the lighting somehow? Changing the default material? Adding shadows?

**Comment by garrlker** [comment]
I can hit this one as well if you don't mind.

**Comment by Peque** [comment]
Of course. Keep in mind that this is a subjective thing. I may not like the new lighting enough to merge the changes.

So try to share your results/ideas early and often to discuss them before spending too much work on something that may not be merged. :wink:

Ideally, changes could be inspired by some existing, well-known CAD tool, which may already have better looks.
